lib/IDE/CodeCompletionCache.cpp

```diff
@@ -209,7 +209,6 @@ static bool readCachedModule(llvm::MemoryBuffer *in,
     auto numBytesToErase = static_cast<unsigned>(*cursor++);
     auto chunkIndex = read32le(cursor);
     auto moduleIndex = read32le(cursor);
-    auto sourceFilePathIndex = read32le(cursor);
```

Comment:
`onDiskCompletionCacheVersion` wasn't bumped for the original change, but should it have been? In which case, should it be bumped now?

Reply:
Heh, given that the version is "2" I suspect we've forgotten to update it multiple times now.  Yes, it should probably be bumped.

Reply:
Thanks for noticing! I bumped it to "3"